src/frontend/src/flows/displaySingleDeviceWarning.ts

```diff
@@ -29,34 +29,35 @@ const pageContent = () => html`
     ${warningIcon}
     <h1 id="warningHeading">Warning</h1>
     <p>
-      If you have your Internet Identity tied to only one device and you lose
-      that one device, you will be locked out.
+      If you have only one device attached to an identity anchor and lose that
+      device, you will no longer be able to use that identity anchor.
     </p>
     <p>
       Additionally, on iOS and MacOS, clearing your Safari browser data will
-      <em>delete</em> your logins from the device.
+      <em>delete</em> your authentication keys from the respective device,
+      disabling it.
     </p>
     <p>
-      As a best practice we always recommend you add multiple devices to your
-      Internet Identity and at <em>least</em> one actual security key. You can
-      see how to add more devices here:
+      As a best practice we always recommend you add multiple devices to an
+      Identity Anchor and at <em>least</em> one portable authenticator (usb
```

Comment:
Nit: maybe use 'USB' in upper case letters?

Reply:
```suggestion
      Identity Anchor and at <em>least</em> one portable authenticator (USB
```